.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CARGO_BUILD_TARGET: ${{ matrix.target }}
-      CARGO_BINSTALL_LOG_LEVEL: debug
+      BINSTALL_LOG_LEVEL: debug
```

Comment:
Hmm I think I'd missed that last PR... let's remove this line entirely, it's already taken care of in the justfile

Reply:
Ok, I will open another PR for this.